Components/Css/cart.css

```diff
@@ -0,0 +1,54 @@
+.cart-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 3rem;
+}
+
+.horizontal-card {
+    margin-right: 3rem;
+    height: 13rem;
+    box-shadow: 5px 10px 18px #888888;
+    border: 2px solid blue;
+}
+
+.btn-quantity {
+    display: flex;
+    justify-content: left;
+    width: 21rem;
+    margin-bottom: 5rem;
+}
+
+.increment-btn {
+    width: 2rem;
+    height: 2rem;
+    margin-left: 2rem;
+    margin-bottom: 1rem;
+    background-color: white;
+    color: black;
+    border: 2px solid blue;
```

Comment:
create separate color variables.so, that you can change color whenever you want at multiple places